Gruntfile.js

```diff
@@ -144,6 +144,7 @@ module.exports = function(grunt) {
                     "editor/js/ui/tab-info.js",
                     "editor/js/ui/tab-config.js",
                     "editor/js/ui/palette-editor.js",
+                    "editor/js/ui/gitConfig.js",
```

Comment:
I want to keep all the projects related code together. I think this should be `projectUserSettings.js`